README.md

```diff
@@ -33,10 +33,16 @@ To start using **MacKernelSDK** do the following steps:
     ```
 
     Otherwise Xcode will link to the original libkmod.
+    
+5. Add the Headers folder to the **Header Search Paths**.
```

Comment:
That should not be needed for kernel projects. Could you clarify the change?

Reply:
I feel it would generally be easier for the developer if MacKernelSDK is in the Header search path as the APIs could be include more directly. I could remove it if this is not required, though.

Reply:
System and SDK headers are different from local headers as you connect them via `<>`. Better not to pollute the namespace really.

Reply:
OK I see. I will remove it.